.circleci/config.yml

```diff
@@ -17,6 +17,9 @@ jobs:
         - restore_cache:
             keys:
               - pip-cache-0
+
+        - restore_cache:
+            keys:
```

Comment:
I had some issues with debugging so had to remove it at some point, just something to keep in mind

Reply:
Ok, thanks for the heads-up. But even in case we have to remove this again, we still have #98 which should save quite some time compared to the current situation.